proto/ibc/core/channel/v2/query.proto

```diff
@@ -34,8 +34,8 @@ message QueryChannelRequest {
 
 // QueryChannelRequest is the response type for the Query/Channel RPC method
 message QueryChannelResponse {
```

Comment:
Q: how deep with the renaming do we want to go? Could this be called `QueryChannelEndResponse` (and the same for all the others?) 

Do we want to rename params in functions from `channel` to `channelEnd`?